libs/ebpfnetsh/programs.cpp

```diff
@@ -146,6 +145,40 @@ handle_ebpf_add_program(
     return ERROR_SUCCESS;
 }
 
+static DWORD
+_unpin_program_by_id(ebpf_id_t id)
+{
+    ebpf_result_t result;
```

Comment:
why cant this be looked up from _ebpf_id_table ? Why such convoluted approach of enumerating objects by name, then getting info and finally comparing Id ?

Reply:
this is what bpftool does (via getmntent()).  Point is have to enumerate all pinned names.

Reply:
oh - because a single object can be pinned to several pin paths. Got it. If not already present, can you put this is comment somewhere?

Reply:
Right, will do in next PR